build.ps1

```diff
@@ -695,8 +692,30 @@ function Consolidate-RedistInstall($Arch)
 {
   if ($ToBatch) { return }
 
-  Remove-Item -Force -Recurse $($Arch.RedistInstallRoot) -ErrorAction Ignore
-  Copy-Dir "$($Arch.SDKInstallRoot)\usr\bin" "$($Arch.RedistInstallRoot)\usr"
+  if ($Arch -eq $HostArch)
+  {
+    $ProgramFilesName = "Program Files"
+  }
+  elseif ($Arch -eq $ArchX86)
+  {
+    $ProgramFilesName = "Program Files (x86)"
+  }
+  elseif (($HostArch -eq $HostArm64) -and ($Arch -eq $ArchX64))
+  {
+    # x64 programs actually install under "Program Files" on arm64,
+    # but this would conflict with the native installation.
+    $ProgramFilesName = "Program Files (x64)"
```

Comment:
Interesting, how does Microsoft handle the conflict?  What if you need/want both versions?

Reply:
I haven't found anything official but it seems like this is not a scenario that they went through extra lengths to support.

https://stackoverflow.com/questions/74402454/what-is-each-program-files-folder-purpose-in-arm64-architecture